discord/webhook/async_.py

```diff
@@ -1162,16 +1162,16 @@ async def send(
         Forbidden
             The authorization token for the webhook is incorrect.
         TypeError
-            You specified both ``embed`` and ``embeds`` or ``file`` and ``files``
+            You specified both ``embed`` and ``embeds`` or ``file`` and ``files``.
         ValueError
-            The length of ``embeds`` was invalid
+            The length of ``embeds`` was invalid.
         InvalidArgument
             There was no token associated with this webhook.
 
         Returns
         ---------
         Optional[:class:`WebhookMessage`]
-            The message that was sent.
+            The message that was sent, if ``wait`` is ``True``.
```

Comment:
This is already mentioned above. Not sure why it'd be repeated, but if so it'd have to be like this;
```suggestion
            If ``wait`` is ``True`` then the message that was sent, otherwise ``None``.
```